src/ray/raylet/agent_manager.h

```diff
@@ -32,6 +33,7 @@ typedef std::function<std::shared_ptr<boost::asio::deadline_timer>(std::function
 class AgentManager : public rpc::AgentManagerServiceHandler {
  public:
   struct Options {
+    ClientID node_id;
```

Comment:
const?

Reply:
As far as. I know, node id shouldn't be changed until raylet is dead.

Reply:
Good suggestion. I will make the ClientID const.